### ISSUE TITLE
Accessibility for MobileNotifier and using unlock sound for notification of new events

### ISSUE THREAD

**Issue by ghost** [peterhajas/MobileNotifier#84]
yesterday I read about MobileNotifier and I installed it. Now I want to make a few suggestions for this great approach.
I'am blind and so I use the iPhone with Apples build in screen reader VoiceOver. You can find some information about VoiceOver at http://www.apple.com/accessibility/iphone/vision.html. I already tryed several notifier solutions but most of them were inaccessible for VoiceOver. Especially the added status bar icons are not visible for the screen reader.

The current situation: the lock screen notifier isn't accessible now but I see the notification list in the app switcher and I find that it's a great place for that. If I get a sms, I also can read it. Above this list I found an unlabeled button.

In the following I want to describe, which features I would need. Maybe it's possible for you to implement some of them. At the moment the biggest issue for me is, that I often don't know that I got an sms or missed a call. This happens cause there is no notification, I must look at the respectiv icons on the home screen. For example if I got a missed phone call I touch the phone symbol and VoiceOver reads out something like "one missed call". But that's annoying cause at least I must observe the phone, message and mail app. My idea is, that you could take the unlock sound to signalize new events. At the moment I use the standard sound to unlock. For example if I got a message, this sound could change to another one so when I unlock my device I will directly be informed about this event. I could imagine that it would be necessary that the user can choose the sound changing events. I only need this for phone calls and mesages, if you would change the sound for push messages you would never hear the original one. Maybe you also could implement more than one sound. It's obvious that the usage of these alternative sounds should be optional cause I could imagine that it would be annoying for most people.
I often wrote about phone calls, is it possible to get them into the list of events too? I saw that also others requested this feature.
At the moment I must open every new message seperatly to mark them as read also if I already know that it's only advertisement. Maybe you could add a "mark as read" button at the left or right of the message or it's done via a long finger tip or a double click.

If you want to try out VoiceOver on your own you find it under "settings" -> "general" -> "accessibility" -> "VoiceOver" (iPhone 3gs and 4 only). But beware, the iOS gestures will completely change, look at the iPhone manual for more information (should be chapter 25).
The folloing links show how to make iPhone apps accessible for screen reader users:
http://developer.apple.com/technologies/ios/accessibility.html
http://developer.apple.com/library/ios/#documentation/UserExperience/Conceptual/iPhoneAccessibility/Introduction/Introduction.html

Thank you in advance for your work
Best regards
